examples/cull-idle/cull_idle_servers.py

```diff
@@ -254,28 +263,27 @@ def handle_user(user):
             inactive = age
 
         should_cull = (inactive is not None and
-                       inactive.total_seconds() >= inactive_limit)
-        if should_cull:
-            app_log.info(
-                "Culling user %s (inactive for %s)",
-                user['name'], inactive)
+                       inactive.total_seconds() >= user_inactive_limit)
 
-        if max_age and not should_cull:
-            # only check created if max_age is specified
+        if user_max_age and not should_cull:
+            # only check created if max_user_age is specified
             # so that we can still be compatible with jupyterhub 0.8
             # which doesn't define the 'started' field
-            if age is not None and age.total_seconds() >= max_age:
+            if age is not None and age.total_seconds() >= user_max_age:
                 app_log.info(
-                    "Culling user %s (age: %s, inactive for %s)",
-                    user['name'], format_td(age), format_td(inactive))
+                    "Culling user %s (age: %s, inactive for %s vs user_max_age %s)",
```

Comment:
I would remove `vs` since it confuses Perhaps `"Culling user %s (age: %s, inactive for: %s, user_max_age limit: %s)"`